utils/postData.js

```diff
@@ -0,0 +1,18 @@
+const axios = require('axios');
+const config = require('../controllers/constants');
+
+async function postData(allMatchData) {
+  try {
+    const endpoint = `${config.serverUrl}/api/todo`;
+    const response = await axios.post(endpoint, { matches: allMatchData });
+    console.log(`Data sent to ${endpoint}:`, response.data);
```

Comment:
api calls are not util functions. place this in the correct location

Reply:
do not call another api responsible for handling the db entry. handle it in the api you are creating